File: Orientado a Objeto/Relatorio 6 - Javascript/cachorro.js
import { Animal } from './animal.js';

class Cachorro extends Animal {
    constructor(nome, idade, especie, raca) {
        super(nome, idade, especie);
        this.raca = raca;
    }

    printInfo() {
        console.log(`
            ${this.printInfo()}\n
            Raça: ${this.raca}`
        );
    }
}

File: Orientado a Objeto/Relatorio 8 - C#/cachorro.cs
using System;

class Cachorro {
    public string nome;
    public int idade;

    public Cachorro(string nome, int idade) {
        this.nome = nome;
        this.idade = idade;
    }

    public void showDogName() {
        Console.WriteLine("O nome do cachorro é: " + nome);
    }

    public void showIdade() {
        Console.WriteLine("A idade do cachorro é: " + idade);
    }
}